app/lib/preload.js

```diff
@@ -182,3 +192,15 @@ function createBackend(ipcRenderer, platform) {
     return platform;
   }
 }
+
+function handleInPreload(event, ...args) {
+  if (event === 'file:get-path') {
+    try {
+      const path = webUtils.getPathForFile(args[0]);
+
+      return path || null;
```

Comment:
This is hard to test as we need a real file on disk (or mock webutils which defies the test).